src/secure_cookie/session.py

```diff
@@ -273,7 +272,7 @@ def save(self, session):
             f.close()
 
         try:
-            rename(tmp, fn)
+            os.rename(tmp, fn)
```

Comment:
This should be `os.replace`, that's the function that matches the `posixemulation` `rename` function.

Reply:
Oops. That was definitely a typo. Nice catch 👍🏻